README.md

```diff
@@ -1,6 +1,4 @@
 # Hello-Microverse
-A Simple Microverse Project
+A Simple Microverse Project that is made to help me show the use of Linters
```

Comment:
- [ ] As stated the project requirement Kindly Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.